src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTextUI.java

```diff
@@ -1029,10 +1029,9 @@ public Dimension getMaximumSize(JComponent c) {
 
 
     /**
-     * Gets the allocation to give the root View.  Due
-     * to an unfortunate set of historical events this
-     * method is inappropriately named.  The Rectangle
-     * returned has nothing to do with visibility.
+     * Gets the allocation to give the root View.
+     * The Rectangle returned has nothing to do with visibility but
+     * the method is thus named to preserve backward compatibility.
```

Comment:
```suggestion
     * the method name is preserved for backward compatibility.
```
If method name is retained from past then this sentence would be right ?

Reply:
I agree. The above rephrasing sounds clearer.